redux-todomvc/containers/App.js

```diff
@@ -31,6 +31,7 @@ function mapStateToProps(state) {
 }
 
 function mapDispatchToProps(dispatch) {
+  marionetteDispatch(dispatch, window.Backbone, window._);
```

Comment:
You have to pass in Backbone, Marionette, and underscore because your app old app may use globals, requirejs, webpack, etc. and we're not going to judge.